xml/System.Security.Cryptography/RSAOpenSsl.xml

```diff
@@ -12,7 +12,15 @@
   <Interfaces />
   <Docs>
     <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+> [!NOTE]
+> The <xref:System.Security.Cryptography.RSAOpenSsl> class is a CNG implementation for Linux and macOS platforms. In a cross-platform scenario, you should use the platform agnostic <xref:System.Security.Cryptography.RSA.Create%2A?displayProperty=fullName> to create an instance of the default implementation for the platform where your code will run.
```

Comment:
RSAOpenSsl isn't "a CNG implementation".  It's a specific implementation of RSA, using OpenSSL's libcrypto, and as such can do platform interop (for example, to make use of a smartcard/HSM).